libraries/DataFlash/DataFlash_File.h

```diff
@@ -103,6 +103,8 @@ class DataFlash_File : public DataFlash_Backend
     ByteBuffer _writebuf;
     const uint16_t _writebuf_chunk;
     uint32_t _last_write_time;
+    //
+    DataFlash_Class *_dataflash;
```

Comment:
This is definitely not right.

Check the way we do disarm-rotate on the file backend (`_front`)